cache/cache.go

```diff
@@ -78,9 +83,8 @@ func NewCache(l log.Logger) (*Cache, error) {
 	return cache, nil
 }
 
-// Save marshalls and stores the data a Cache is holding into disk as a JSON
-// file.
-func (cache *Cache) Save() error {
+// Save persists all the data in the cache.
+func (cache *cacheImpl) Save() error {
```

Comment:
how about `cache *cacheImpl` => `c *Cache` ?

Golang methods receivers tend to be 1or2 chars long